### PR TITLE
LPS-143108 Create GET UserGroup for User Group Headless

### PR DESCRIPTION
**Pull request by erickmonteiroo**
Rebased on [LPS-143045](https://issues.liferay.com/browse/LPS-143045)

### CODE REVIEW

File: modules/apps/headless/headless-admin-user/headless-admin-user-impl/rest-openapi.yaml [hunk 1]
```diff
@@ -962,6 +962,35 @@ components:
...
+                    description:
+                        The description of the user group.
+                    type: string
+                id:
+                    description:
+                        The id of the user group.
+                    format: int64
+                    readOnly: true
+                    type: integer
+                name:
```
Comment (drewbrokke) [inline modules/apps/headless/headless-admin-user/headless-admin-user-impl/rest-openapi.yaml:979]: I don't think it's possible to create a UserGroup without a name, right? If so, this property should be `required`.

File: modules/apps/headless/headless-admin-user/headless-admin-user-test/src/testIntegration/java/com/liferay/headless/admin/user/resource/v1_0/test/UserGroupResourceTest.java [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+import com.liferay.arquillian.extension.junit.bridge.junit.Arquillian;
+
+import org.junit.Ignore;
+import org.junit.runner.RunWith;
+
+/**
+ * @author Javier Gamarra
+ */
+@Ignore
```
Comment (drewbrokke) [inline modules/apps/headless/headless-admin-user/headless-admin-user-test/src/testIntegration/java/com/liferay/headless/admin/user/resource/v1_0/test/UserGroupResourceTest.java:25]: We should un-ignore this test is possible.

**Review by drewbrokke — CHANGES_REQUESTED** [review]
@erickmonteiroo these changes are looking good. We need to adjust the schema a little though. IMO we should focus on getting the schema and basic `GET` endpoint right, then the rest of the endpoints will fall into place.

**Comment by erickmonteiroo** [comment]
ci:test:relevant

**Comment by erickmonteiroo** [comment]
ci:test:relevant

**Comment by erickmonteiroo** [comment]
ci:test:relevant

**Comment by erickmonteiroo** [comment]
ci:test:relevant

**Comment by drewbrokke** [comment]
@erickmonteiroo looks like a bunch of merge conflicts (no surprise). Please rebase and force-push and I'll give it a review.

**Comment by erickmonteiroo** [comment]
@drewbrokke rebased

**Review by drewbrokke — APPROVED** [review]
Looks good @erickmonteiroo!

**Comment by drewbrokke** [comment]
ci:forward

**Comment by drewbrokke** [comment]
ci:reevaluate:1307401_1283

**Comment by erickmonteiroo** [comment]
ci:test:relevant

**Comment by epark18** [comment]
@erickmonteiroo None of the failures are unique to this pull. [Case Results](https://testray.liferay.com/home/-/testray/case_results?testrayBuildId=1588997830&testrayRunId=1588998696) | A lot of the tests are failing across a lot of other PRs at the moment from the same issues. Additionally, `AutoTaggingFramework#CanAddAutoTagAfterEnablingOpenNLP` failed due to a connection issue.
The comparison failures `org.junit.ComparisonFailure: expected:<[Not Found]> but was:<[Internal Server Error]>
	at org.junit.Assert.assertEquals(Assert.java:117)` from earlier checks are no longer present.

**Review by drewbrokke — APPROVED** [review]
ci:forward

**Comment by erickmonteiroo** [comment]
ci:forward

**Pull request closed without merging**